src/main/scala/org/ergoplatform/network/ErgoNodeViewSynchronizer.scala

```diff
@@ -77,6 +77,26 @@ class ErgoNodeViewSynchronizer(networkControllerRef: ActorRef,
     }
   }
 
+  //todo: pull back to Scorex
+  override protected def checkDelivery: Receive = {
```

Comment:
Please describe why this method require override, and why it should be pulled back to Scorex?